library/src/main/java/com/meisolsson/githubsdk/model/ClientErrorResponse.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015 Henrik Olsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.meisolsson.githubsdk.model;
+
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.meisolsson.githubsdk.core.FormattedTime;
+import com.google.auto.value.AutoValue;
+import com.squareup.moshi.Json;
+import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.Moshi;
+
+import java.util.Date;
+import java.util.List;
+
+@AutoValue
+public abstract class ClientErrorResponse implements Parcelable {
```

Comment:
Not in the SDK, no. I use it in my error handling: https://github.com/slapperwan/gh4a/blob/rx/app/src/main/java/com/gh4a/ApiRequestException.java#L29 ... I felt it should live here because it's a GitHub data structure.